src/main/java/org/prebid/server/bidder/rediads/RediadsBidder.java

```diff
@@ -0,0 +1,181 @@
+
+        final BidRequest outgoingRequest = modifyRequest(request, modifiedImps, accountId);
+        final String endpointUrl = resolveEndpointUrl(endpoint);
+        final HttpRequest<BidRequest> httpRequest = BidderUtil.defaultRequest(outgoingRequest, endpointUrl, mapper);
+
+        return Result.of(Collections.singletonList(httpRequest), errors);
+    }
+
+    private ExtImpRediads parseImpExt(Imp imp) {
+        try {
+            return mapper.mapper().convertValue(imp.getExt(), TYPE_REFERENCE).getBidder();
+        } catch (IllegalArgumentException e) {
+            throw new PreBidException("Invalid imp.ext for impression " + imp.getId());
+        }
+    }
+
+    private Imp modifyImp(Imp imp, ExtImpRediads extImp) {
+        final ObjectNode modifiedExt = imp.getExt().deepCopy();
+        modifiedExt.remove("bidder");
+        modifiedExt.remove("prebid");
+        return imp.toBuilder()
+                .tagid(StringUtils.defaultIfBlank(extImp.getSlot(), imp.getTagid()))
+                .ext(modifiedExt)
+                .build();
+    }
+
+    private BidRequest modifyRequest(BidRequest request, List<Imp> imps, String accountId) {
+        final Site site = request.getSite();
+        final App app = request.getApp();
+        return request.toBuilder()
+                .site(site != null ? modifySite(site, accountId) : null)
+                .app(site == null && app != null ? modifyApp(app, accountId) : app)
+                .imp(imps)
+                .build();
+    }
+
+    private static Site modifySite(Site site, String accountId) {
+        final Publisher originalPublisher = site.getPublisher();
+        final Publisher newPublisher = originalPublisher != null
+                ? originalPublisher.toBuilder().id(accountId).build()
+                : Publisher.builder().id(accountId).build();
+        return site.toBuilder().publisher(newPublisher).build();
+    }
+
+    private static App modifyApp(App app, String accountId) {
+        final Publisher originalPublisher = app.getPublisher();
+        final Publisher newPublisher = originalPublisher != null
+                ? originalPublisher.toBuilder().id(accountId).build()
+                : Publisher.builder().id(accountId).build();
+        return app.toBuilder().publisher(newPublisher).build();
+    }
+
+    private String resolveEndpointUrl(String endpoint) {
+        if (StringUtils.isBlank(endpoint)) {
+            return endpointUrl;
+        }
+
+        try {
+            final URL originalUrl = new URL(endpointUrl);
+            final String originalHost = originalUrl.getHost();
+            final String[] hostParts = originalHost.split("\\.");
+            hostParts[0] = endpoint;
+            final String newHost = String.join(".", hostParts);
+            return endpointUrl.replace(originalHost, newHost);
+        } catch (MalformedURLException e) {
+            throw new PreBidException("Failed to parse endpoint URL: " + endpointUrl);
+        }
+    }
```

Comment:
Let's change this a bit:
- in `rediads.yaml` replace endpoint with `https://{{SUBDOMAIN}}.rediads.com/openrtb2/auction`
- replace macros with incoming value or `bidding` as a default